src/manage/advanced.js

```diff
@@ -15,7 +15,243 @@ class Advanced extends React.Component {
+                placeholder=""
+                length="3"
+                description="Allowed Applications / APIs are clients that will be allowed to make delegation request. By default, all your clients will be allowed. This field allows you to enter specific client ids. You can specify multiple IDs by comma-separating them or one by line."
+              />
+
+              <Form.Select
+                label="JsonWebToken Signature Algorithm"
+                readOnly
+                options={[
+                  { text: 'RS256', value: '1', defaultSelected: true },
+                  { text: 'Second option', value: '2' },
+                  { text: 'Third option', value: '3' },
+                  { text: 'Fourth option', value: '4' }
+                ]}
+              />
+              <Form.Switch
+                label="OIDC Conformant"
+                on
+                description={
+                  <span>
+                    Clients flagged as OIDC Conformant will strictly follow the OIDC specification.
+                    Turning on this flag can introduce breaking changes to this client. If you have
+                    any questions you can <a href="link">contact support</a>.
+                  </span>
+                }
+              />
+              <Form.TextInput
+                label="Cross-Origin Verification Fallback"
+                type="text"
+                placeholder="https://domain.tld/path"
+                description="Location URL for the page that will be rendered inside an iframe to perform the token verification when third party cookies are not enabled in the browser. Must be in the same domain where the embedded login form is hosted and must have a `https` scheme."
+              />
+            </Form.FieldSet>
+            <Form.Actions primaryAction={{ label: 'Save Changes', method: this.save }} />
+          </Form>
+        </Well>
+        <Well>
+          <Form>
+            <Form.FieldSet label="Grant Types">
+              <p>
+                Notification: Using Password or MFA grant types with public clients is not
+                recommended. To use the Client Credentials grant you have to set a Token Endpoint
+                Auth Method other than "none". See our documentation for more information.
+              </p>
+              <br />
+              <p>Grants go here... </p>
+            </Form.FieldSet>
+          </Form>
+        </Well>
+        <Well>
+          <Form>
+            <Form.FieldSet label="WS-Federation">
+              <p>Code block goes here...</p>
+            </Form.FieldSet>
+            <Form.Actions primaryAction={{ label: 'Save Changes', method: this.save }} />
+          </Form>
+        </Well>
+        <Well>
+          <Form>
+            <Form.FieldSet label="Certificates">
+              <Form.TextArea
+                label="Signing Certificate"
+                code
+                readOnly
+                value="-----BEGIN CERTIFICATE-----
```

Comment:
Where did this certificate come from? The accuracy of the format worries me 😅

Reply:
I have my own testing account, where I've changed some random letters, but probably not enough. I'll do some more faking so that it doesn't look so real.